src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each SecondaryNavLink can take active, onNavClick as props, and {tabText}
+ * as children.
+ * `<SecondaryNavLink key={key} active={ifActive} onNavClick={() => {}} href={herf}>`
+ * `{tabText}`
+ * `</SecondaryNavLink>`
+ */
+const SecondaryNav = ({ buttons, className }) => {
+  const navClass = classes('SecondaryNav', className);
+  return (
+    <nav className={navClass} aria-labelledby="Secondary Navigation">
```

Comment:
remove the `aria-albelledby` as it turns out I misunderstood it's usage according to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute